back-end/server.js

```diff
@@ -16,7 +16,7 @@ const http = require('http');
     const VersionChecker = require('./domain/cimonitor/VersionChecker');
     const router = require('./routes');
 
-    const Config = await ConfigLoader.getConfig();
+    const Config = ConfigLoader.getConfig();
```

Comment:
This should still await right? Else you potentially start services without the right config when it needs to be load from Firebase

Reply:
Sure, I believe this was the double await we found earlier but it's needed nonetheless. BACK!